istioctl/docker/Dockerfile.istioctl

```diff
@@ -0,0 +1,20 @@
+# BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
+ARG BASE_DISTRIBUTION=default
```

Comment:
I don't think we need to ship two variants of this since its just used in CI. It probably doesn't make much sense to not have a shell in this image?

Reply:
Good point. I removed the argument/logic and just rely on the base image now.